dev-env/windows/manifests/nodejs.json

```diff
@@ -1,18 +1,18 @@
 {
     "homepage": "https://nodejs.org",
     "description": "As an asynchronous event driven JavaScript runtime, Node is designed to build scalable network applications. (Long Term Support)",
-    "version": "14.18.3",
+    "version": "16.18.1",
```

Comment:
Should the major here be 18?

Reply:
I'm still trying to figure this out .The end goal is that it works for our users on Node 18, not necessarily that we build with Node 18. Navigator doesn't build on Node 18 at the moment on CI (but does on my machine 🤷), so I'm trying things.

That's why it's still a draft.